--- vendor/fltk-sys/build.rs.orig	2020-11-16 11:01:13.258311445 +0100
+++ vendor/fltk-sys/build.rs	2020-11-16 11:00:03.214409132 +0100
@@ -214,9 +214,19 @@ fn main() {
 
     println!(
         "cargo:rustc-link-search=native={}",
+        out_dir.join("lib64").display()
+    );
+
+    println!(
+        "cargo:rustc-link-search=native={}",
         out_dir.join("lib").join("Release").display()
     );
 
+    println!(
+        "cargo:rustc-link-search=native={}",
+        out_dir.join("lib64").join("Release").display()
+    );
+
     if !cfg!(feature = "fltk-shared") {
         println!("cargo:rustc-link-lib=static=cfltk");
     } else {
